LaTeX assignment Klaudia Bielawa.txt

```diff
@@ -0,0 +1,62 @@
+﻿\documentclass{article}
+\usepackage[utf8]{inputenc}
+\usepackage{tikz}
+\usepackage{phonrule}
+\usepackage{forest}
+\usepackage{tipa}
+\usepackage{amsmath}
+
+\title{LaTeX assignment}
+\author{Klaudia Bielawa}
+\date{June 2019}
+\usepackage[style=authoryear]{biblatex}
+
+\begin{document}
+\maketitle
+
+\section{Section 1}
+My favorite topics in linguistics:
+\begin{itemize}
+  \item Semantics
+  \item Phonology
+  \item Morphology
+\end{itemize}
+
+\section{Section 2}
+\begin{IPA}
+
+maj nem ɪz klɔdiə aj lajv ən gədænsk || aj hæv twɛnti fajv jɪrz old | aj wərk ən dʒəmp sɪti ɪt ɪz e træmpəlin park || aj lajk kantækt wɪð awər gɛsts | aj ləv maj ko wərkərz ðe ar friki aj ləv bʊks ðæt ar rɪtən baj stivən kɪŋ|
```

Comment:
Dear Klaudia, 

this does not look like phonetic transcription... :( Please look up **each word** in a phonetic dictionary and later try to find correspondences in tipa documentation.